test/safe/discovery.test.ts

```diff
@@ -58,6 +96,79 @@ describe("Discovery Queue Test", () => {
 		expect(states.length).toBe(2);
 	}
 
+	// eslint-disable-next-line jest/expect-expect
```

Comment:
I'd probably put that at the top of the file instead of per test if you're going to repeat it